pkg/util/util.go

```diff
@@ -69,6 +69,14 @@ func logPanic(r interface{}) {
 	glog.Errorf("Recovered from panic: %#v (%v)\n%v", r, r, callers)
 }
 
+// Min will return the smaller of the 2 given integers.
```

Comment:
Actually can I keep the function in to reduce duplication, since I'm using it in the unittests as well? Also I feel like Min(a,b int) is the most common use case, so it might be useful?